### ISSUE TITLE
Am I missing something here? There seems to be no angular 2.0 ?

### ISSUE THREAD

**Issue by BrewDawg** [arvindr21/ng2do-mean-app#2]

**Comment by gh0st** [comment]
No, I don't think you're missing anything. I'm thinking this was written in the early ages of Angular2 when it hadn't fully implemented. _Or_, this Angular2 app was written in JavaScript, as opposed to TypeScript. As of today, there are three different languages that Angular2 apps can be written in. If you visit the [Angular Quickstart](https://angular.io/docs/dart/latest/quickstart.html) they go over how to write apps in TypeScript, JavaScript, and even Dart.
